### ISSUE TITLE
Missing Hashrate on Proxy window

### ISSUE THREAD

**Issue by i8088** [kralverde/ravencoin-stratum-proxy#13]
I have 1250 Mh/s total and  8 worker . 
But Proxy say ; 120-300 mh /s  ..  is it normal ?
![rvnhash](https://user-images.githubusercontent.com/33055151/191947942-9501ea6b-b2d9-418d-a7cb-04a1b2fc6c2e.JPG)

**Comment by kralverde** [comment]
yeah the hashrate currently only reports for a single miner, a pr is in the works for the total hashrate

**Comment by kralverde** [comment]
@Seal-Clubber

**Comment by Seal-Clubber** [comment]
[Hashrate](https://github.com/Seal-Clubber/ravencoin-stratum-proxy/tree/Hashrate) branch doesn't use new libs, It works as long as you don't restart your miner since that causes it to send out a new ClientID, and the old ID has no mechanism to remove it after some inactivity.

OR use [Hashrate-dictlib](https://github.com/Seal-Clubber/ravencoin-stratum-proxy/tree/Hashrate-dictlib) branch which uses the [Expiring-dict](https://pypi.org/project/expiring-dict/) library, it removes inactive ClientIDs every 2minutes. I did some quick test and it seems to work, but would love to have someone test it with a longer period of time. (TODO still need to add the sha256 for the requirments.txt)

**Comment by Seal-Clubber** [comment]
#15

**Comment by kralverde** [comment]
https://github.com/kralverde/ravencoin-stratum-proxy/pull/17
